tools/build_chain.sh

```diff
@@ -1286,6 +1294,59 @@ check_bin()
     echo "Binary check passed."
 }
 
+prepare_ca(){
+    dir_must_not_exists ${output_dir}/chain
+    if [ -z "$ca_path" ]; then
+        echo "Generating CA key..."
+        gen_chain_cert ${output_dir}/chain >"${logfile}" 2>&1 || exit_with_clean "openssl error!"
+        mv ${output_dir}/chain ${output_dir}/cert
+    else
+        echo "Use user specified ca key from ${ca_path}"
+        mkdir -p ${output_dir}/cert
+        cp "${ca_key}" ${output_dir}/cert
+        cp "${ca_crt}" ${output_dir}/cert
+    fi
+    ca_key="${output_dir}/cert/ca.key"
+    mv cert.cnf ${output_dir}/cert/
+    if [ "${use_ip_param}" == "false" ];then
+        for agency_name in ${agency_array[*]};do
+            if [ ! -d ${output_dir}/cert/${agency_name} ];then 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5331982)